gui/main.py

```diff
@@ -101,12 +104,36 @@ def post(self):
 
 
 class DiscoverBotsHandler(tornado.web.RequestHandler):
+    """
+    Listens for bot discoverability.
+    """
     def post(self):
         discovered = BaseStation().get_bot_manager().get_all_discovered_bots()
+        print(discovered)
```

Comment:
Is this really necessary?

Reply:
Print statement feedback for the terminal. Added to print statement for context.